Makefile.PL

```diff
@@ -198,6 +198,16 @@ if ($^O eq 'VMS') {
 }
 $cflags .= " -DDBD_MYSQL_NO_CLIENT_FOUND_ROWS" if $opt->{'nofoundrows'};
 $cflags .= " -g ";
+
+unless ($^O eq 'MSWin32') {
+  if ($Config{usethreads} || $Config{osname} =~ /darwin|linux/i || $Config{libs} =~ /-l?pthread\b/ || $Config{ldflags} =~ /-l?pthread\b/) {
```

Comment:
I'm not sure this condition is correct, as it will assume that any mac or linux install has pthreads which is not true?

Reply:
Though I see that is also done in the condition I linked to, seems strange.

Reply:
Yes, that's why I did it that way. I can imagine that OSX systems generally have pthreads and I suppose the thinking for Linux is that pretty much any system not older than 2 decades will have pthreads there too.